code-challenges.test.js

```diff
@@ -52,5 +151,39 @@ const numbersToAdd2 = [0, 7, -8, 12]
 const numbersToAdd3 = []
 // Expected output: []
 
+describe("sum", () => {
+    const numbersToAdd1 = [2, 4, 45, 9]
+    const numbersToAdd2 = [0, 7, -8, 12]
+    const numbersToAdd3 = []
+    it("takes in an array and returns an array of the accumulating sum", () => {
+        expect(sum(numbersToAdd1)).toEqual([2, 6, 51, 60])
+        expect(sum(numbersToAdd2)).toEqual([0, 7, -1, 11])
+        expect(sum(numbersToAdd3)).toEqual([])
+    })
+})
```

Comment:
Your test looks great. A small improvement would be to put the test variables inside the `it` block so they share they are defined and referenced in the same scope.